### PR TITLE
#497 Added new methods

### PR DESCRIPTION
**Pull request by kronicz**

### CODE REVIEW

File: docs/automation/crmscript/reference/CRMScript.Global.Void.yml [hunk 1]
```diff
@@ -808,6 +810,29 @@ items:
...
+  id: 'encodeBase32(Byte[])'
+  so.intellisense: Void.encodeBase32
+  so.version: 10.1.9
+  langs:
+  - crmscript
+  name: 'encodeBase32(Byte[])'
+  nameWithType: Void.encodeBase32(Byte[])
+  fullName: CRMScript.Global.Void.encodeBase32(Byte[])
+  type: Method
+  summary: "\nEncodes bytes to a base 32 string.\n"
```
Comment (digitaldiina) [inline docs/automation/crmscript/reference/CRMScript.Global.Void.yml:824]: use Base32 when referring to the format as a noun, use base-32 (hyphenated) as adjective

Same goes for Base64 / base-64, I suggest to clean up both for consistency

File: docs/automation/crmscript/reference/CRMScript.Global.Void.yml [hunk 2]
```diff
@@ -906,6 +931,32 @@ items:
...
+  fullName: CRMScript.Global.Void.encodeHMACSHA1(Byte[],Byte[])
+  type: Method
+  summary: "\nGenerates and returns a hash from a specified key-value bytearray using the HMAC SHA1 encoding algorithm.\n"
+  remarks: "\nThis can be used to ensure data integrity. The hash cannot be decrypted back.\n"
+  syntax: 
+    content: Byte[] encodeHMACSHA1(Byte[] key, Byte[] value)
+    parameters:
+    - id: key
+      type: CRMScript.Global.Byte[]
+      description: "The key used to creating the hash"
```
Comment (digitaldiina) [inline docs/automation/crmscript/reference/CRMScript.Global.Void.yml:952]: Either "used to create" or "used for creating"

File: docs/automation/crmscript/reference/CRMScript.Global.Void.yml [hunk 3]
```diff
@@ -906,6 +931,32 @@ items:
...
+    - id: value
+      type: CRMScript.Global.Byte[]
+      description: "The value to be hashed."
+    return:
+      type: CRMScript.Global.Byte[]
+      description: "The generated hash"
```
Comment (digitaldiina) [inline docs/automation/crmscript/reference/CRMScript.Global.Void.yml:958]: Missing period at end of sentence. Description in lines 958 and 952.

File: docs/automation/crmscript/reference/CRMScript.Global.Void.yml [hunk 4]
```diff
@@ -906,6 +931,32 @@ items:
...
+  id: 'encodeHMACSHA1(Byte[],Byte[])'
+  so.intellisense: Void.encodeHMACSHA1
+  so.version: 10.1.9
+  langs:
+  - crmscript
+  name: 'encodeHMACSHA1(Byte[],Byte[])'
+  nameWithType: Void.encodeHMACSHA1(Byte[],Byte[])
+  fullName: CRMScript.Global.Void.encodeHMACSHA1(Byte[],Byte[])
+  type: Method
+  summary: "\nGenerates and returns a hash from a specified key-value bytearray using the HMAC SHA1 encoding algorithm.\n"
```
Comment (digitaldiina) [inline docs/automation/crmscript/reference/CRMScript.Global.Void.yml:945]: It's still a *key-value pair*, and 2 arrays are needed. "byte array" is 2 words.

File: docs/automation/crmscript/reference/CRMScript.Global.Void.yml [hunk 5]
```diff
@@ -808,6 +810,29 @@ items:
     return:
       type: CRMScript.Global.Void
   example: 
+- uid: CRMScript.Global.Void.encodeBase32(Byte[])
+  commentId: M:CRMScript.Global.Void.encodeBase32(Byte[])
+  id: 'encodeBase32(Byte[])'
+  so.intellisense: Void.encodeBase32
+  so.version: 10.1.9
```
Comment (digitaldiina) [inline docs/automation/crmscript/reference/CRMScript.Global.Void.yml:817]: Should we hold merge till this version is released?

**Review by digitaldiina — CHANGES_REQUESTED** [review]

File: docs/automation/crmscript/reference/CRMScript.Global.Void.yml (hunk 5 above)
Comment (kronicz) [reply docs/automation/crmscript/reference/CRMScript.Global.Void.yml:817]: Could just merge this now, so we can generate a new intellisense file

**Review by digitaldiina — APPROVED** [review]

**Pull request merged**